Makefile

```diff
@@ -10,7 +10,7 @@ test:
 
 test-integration: up
```

Comment:
@raulb to prevent the pr disorganization I made with the pinecone connector, I'll wait this time for the comment pr to be merged to main before this one get's merged.